test/Frontend/crash-in-user-code.swift

```diff
@@ -1,3 +1,4 @@
+// UNSUPPORTED: static_stdlib
```

Comment:
This is not obvious, could you add a comment about this? (Here and in every other test, please.)

Reply:
Done. Since the explanation is somewhat wordy, I've created a Jira bug for the UNSUPPORTED annotations and referenced to the bug.

Do you think the (unrelated) XFAIL elsewhere needs a comment as well?